keps/NEXT_KEP_NUMBER

```diff
@@ -1 +1 @@
-34
+35
```

Comment:
drop the change to this file, use of a conflict-prone incrementer is no longer part of the kep template

Reply:
Will do.  FWIW, it's still in the template...https://github.com/kubernetes/enhancements/blob/master/keps/0000-kep-template.md
The PR hasn't been merged yet, so the process is confusing :-|

Also renamed the file to drop the KEP number.